pkg/api/pod/util.go

```diff
@@ -1003,6 +1009,9 @@ func dropDisabledPodStatusFields(podStatus, oldPodStatus *api.PodStatus, podSpec
 		dropAllocatedResourcesField(podStatus.ContainerStatuses)
 		dropAllocatedResourcesField(podStatus.InitContainerStatuses)
 		dropAllocatedResourcesField(podStatus.EphemeralContainerStatuses)
+
+		/// Drop pod-level AllocatedResources field
+		podStatus.AllocatedResources = nil
```

Comment:
does this need to be here? `AllocatedResources` is cleared in the code block above (L986-990)

We're about to promote `InPlacePodVerticalScaling` to GA so this entire code block (L992-L1015) will become moot

Reply:
yeah, the above line should be sufficient